src/utils/httpUtils.ts

```diff
@@ -11,3 +13,7 @@ export function getURL(baseURL: string, path: string, orgId?: string): string {
 function routeToGateway(baseURL: string): boolean {
   return baseURL.includes('snykgov.io');
 }
+
+function isValidOrg(orgId?: string): boolean {
+  return Boolean(orgId && ORG_ID_REGEXP.exec(orgId));
```

Comment:
enh: `ORG_ID_REGEXP.test` may be more along the lines of what you are looking for?